common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -88,14 +88,19 @@
     </script>
 </%def>
 
-<%def name="optional_include_mako(file, with_microsite=False)"><%
+<%def name="optional_include_mako(file, with_microsite=False, fallback=None, **kwargs)"><%
```

Comment:
`with_microsite` was renamed to `is_theming_enabled` in another PR, make sure to update this variable name if you are using it. I think it is one of the conflicts github is showing.